### ISSUE TITLE
command line executable doesn't works since 1.6.0

### ISSUE THREAD

**Issue by u3shit** [ruby-prof/ruby-prof#323]
I've tried to `ruby-prof` 1.6.1 on ruby 3.2.1 but I ran into a problem:

```
$ ruby-prof ./baz.rb 
/home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/lib/ruby-prof/printers/abstract_printer.rb:90:in `print_threads': undefined method `threads' for nil:NilClass (NoMethodError)

      @result.threads.each do |thread|
             ^^^^^^^^
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/lib/ruby-prof/printers/abstract_printer.rb:66:in `print'
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:323:in `block in <top (required)>'
/home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:285:in `initialize': unknown keywords: :printer, :min_percent, :file, :exclude, :pre_libs, :pre_execs (ArgumentError)

      @profile = Profile.new(options.to_h)
                             ^^^^^^^^^^^^
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:285:in `new'
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:285:in `run'
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:328:in `<top (required)>'
        from /home/u3/bin/ruby-prof:25:in `load'
        from /home/u3/bin/ruby-prof:25:in `<main>'
/home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:285:in `initialize': unknown keywords: :printer, :min_percent, :file, :exclude, :pre_libs, :pre_execs (ArgumentError)

      @profile = Profile.new(options.to_h)
                             ^^^^^^^^^^^^
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:285:in `new'
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:285:in `run'
        from /home/u3/.gem/ruby/3.2.0/gems/ruby-prof-1.6.1/bin/ruby-prof:328:in `<top (required)>'
        from /home/u3/bin/ruby-prof:25:in `load'
        from /home/u3/bin/ruby-prof:25:in `<main>'
```

`baz.rb` is an empty file, but as far as I can tell it doesn't even get to loading the script. The two last errors are seem to be produced by this change, as the `Profile` constructor no longer ignores unknown keys: https://github.com/ruby-prof/ruby-prof/commit/260904ef544aca6f858b13541c849e19e5b4eff6, I'm not sure about the threading one.
Downgrading to `1.5.0` fixes the problem, so it's not a ruby version problem.

**Comment by cfis** [comment]
Ah, probably a change to keywords. Thanks for the report will take a look. Or happy to take a path too.

**Comment by cfis** [comment]
Thanks for the report, fixed in 1.6.2
